app/code/Magento/InventorySalesApi/Test/_files/stock_website_link.php

```diff
@@ -35,3 +37,7 @@
     $extensionAttributes->setSalesChannels($salesChannels);
     $stockRepository->save($stock);
 }
+
+$indexer = Bootstrap::getObjectManager()->create(IndexerInterface::class);
```

Comment:
Pls use `https://github.com/magento-engcom/msi/blob/develop/app/code/Magento/InventoryIndexer/Test/_files/reindex_inventory.php` fixture in annotations instead of direct call in another fixture

Reply:
Done.